nixos/modules/services/databases/mongodb.nix

```diff
@@ -125,7 +139,7 @@ in
     };
     users.groups.mongodb = lib.mkIf (cfg.user == "mongodb") { };
 
-    environment.systemPackages = [ mongodb ];
+    environment.systemPackages = [ mongosh ];
```

Comment:
I don't think `mongosh` should be added in here.

Reply:
In that case the line should be removed entirely. The `mongodb` package was only useful to access the shell, there is no reason to access the mongodb daemon outside the systemd service.

Reply:
Yes, I would remove it.